.vscode/settings.json

```diff
@@ -19,5 +19,16 @@
 	"**/.cache/**",
 	"**/jetpack_vendor/**/jetpack_vendor/**",
 	"**/jetpack_vendor/**"
-	]
+	],
+	"phpcs.enable": true,
+	"phpcs.executablePath": "vendor/bin/phpcs",
+	"phpcs.ignorePatterns": [
+		"**/.git/**",
+		"**/.svn/**",
+		"**/.hg/**",
+		"**/.cache/**",
+		"**/jetpack_vendor/**",
+		"**/vendor/**",
+	],
+	"prettier.prettierPath": "node_modules/.pnpm/wp-prettier@2.8.5/node_modules/wp-prettier/index.js"
```

Comment:
This may be risky, since that exact path may change as we update the package, and we'll have to remember to edit this line every time.

Looking at my own VS Code config, I do not seem to have `prettierPath` set, and it still works. I'm not sure if I did anything special to make things work :) I only have `configPath` set it seems.

Reply:
Hmm, so it will pick up `prettier` if you have it installed globally, which could be a problem in itself, especially since we want to be using our `wp-prettier` fork and not the original library.

Reply:
If we need a path, `node_modules/.bin/prettier` should work.

Reply:
I think there's an issue with the extension and that version of prettier, can't remember exactly, I'll need to track it down again to get the full context. But the gist is, I tried just pointing it at the module and it doesn't work, pointing it at `index.js` directly is a workaround but gets the job done.

Reply:
So, `tools/js-tools/node_modules/prettier/index.js` then?